external-tracer/src/lib.rs

```diff
@@ -10,6 +10,8 @@ use std::collections::HashMap;
 /// Configuration structure for `geth_utlis::trace`
 #[derive(Debug, Default, Clone, Serialize)]
 pub struct TraceConfig {
+    /// 
+    pub enable_skipping_invalid_tx: Word,
```

Comment:
Why not a bool? Also ideally this is not something know in the tracer I think (unless there's a good reason for it).